From 4a029915c4693b16db4806f078504cc86f34c069 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 21 Jan 2014 10:44:12 +0800
Subject: [PATCH] ENGR00295892-2: ARM: dts: imx6qdl-sabresd: add
 retain-state-suspended property in dts

Add property "retain-state-suspended" in dts.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit aff480769c26ff7f64059f9631b10e8177c4791b)
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 708fe7c..ee07ab9 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -39,6 +39,7 @@
 		charger-led {
 			gpios = <&gpio1 2 0>;
 			linux,default-trigger = "max8903-charger-charging";
+			retain-state-suspended;
 		};
 	};
 
-- 
1.8.0

